Minor Changes to Craft Files

JamesKSPModTester opened the pull request:
Added new cameras to the ship and booster + some other minor changes also I added real size PEZ craft files. With V3.4.8 more has been included check changelog for more information

B16SETC committed:
Minor Changes to Craft Files

JamesKSPModTester commented:
![Screenshot (621)](https://github.com/user-attachments/assets/11bd1636-312d-45f1-b869-15038bc8a31a)
btw here are some screenshots from the cameras

B16SETC committed:
Update CHANGELOG.md

B16SETC committed:
Update CHANGELOG.md

B16SETC committed:
Update CHANGELOG.md

B16SETC committed:
Update CHANGELOG.md

B16SETC committed:
Minor Changes to Craft Files

B16SETC committed:
Disabled the thermal paint on the flaps

B16SETC committed:
Added the changes that Nubro made in the code

B16SETC committed:
added the stuff that Nubro added to the code yesterday

B16SETC committed:
Merge branch 'main' into main

B16SETC committed:
Enabled engine shielding on real size craft files

ghost commented:
I closed this pull request because I gotta do some changes when I finish them I'll reopen it

B16SETC committed:
Added new craft files, edited others and did some coding this time too

so in this update I added all the stock size craft files for the new booster and edited the already existent craft files. All of them. I also did some changes to the telemetry with the booster and ship's orientation. I basically allowed them to rotate whichever way. This update is not full this is a part 1. There is a bug with the ship's orientation and 2 of the craft files aren't included so just be patient for part 2 of the update which will fix everything

B16SETC committed:
Added new craft files, edited others and did some coding this time too

B16SETC committed:
Added new craft files, edited others and did some coding this time too

B16SETC committed:
Update CHANGELOG.md

B16SETC committed:
Update CHANGELOG.md

B16SETC committed:
Update CHANGELOG.md

ghost commented:
update released and so the pull request has been reopened

B16SETC committed:
Added new craft files, edited others and did some coding this time too

B16SETC committed:
release

B16SETC committed:
release
